EventGenerator/src/CaloTBGun_module.cc

```diff
@@ -0,0 +1,78 @@
+// Bertrand
```

Comment:
Add some comments describing what this module does.

Reply:
Sorry, a cut and paste issue. This is a generator to simulate the calo test beam conditions by shooting electrons right in front of the crystals. I want to keep this for a little while, but we might be able to remove it when it will be deprecated. Comment added